drivers/ios-driver/src/nativeMain/kotlin/com/squareup/sqldelight/drivers/ios/SQLiterHelper.kt

```diff
@@ -0,0 +1,227 @@
+package com.squareup.sqldelight.drivers.ios
+
+import co.touchlab.sqliter.Cursor
+import co.touchlab.sqliter.DatabaseConnection
+import co.touchlab.sqliter.DatabaseManager
+import co.touchlab.sqliter.Statement
+import co.touchlab.sqliter.bindLong
+import co.touchlab.sqliter.bindDouble
+import co.touchlab.sqliter.bindString
+import co.touchlab.sqliter.bindBlob
+import co.touchlab.sqliter.getBytesOrNull
+import co.touchlab.sqliter.getDoubleOrNull
+import co.touchlab.sqliter.getLongOrNull
+import co.touchlab.sqliter.getStringOrNull
+import co.touchlab.stately.collections.frozenHashMap
+import co.touchlab.stately.collections.frozenLinkedList
+import co.touchlab.stately.concurrency.AtomicBoolean
+import co.touchlab.stately.concurrency.AtomicReference
+import co.touchlab.stately.concurrency.QuickLock
+import co.touchlab.stately.concurrency.withLock
+import co.touchlab.stately.freeze
+import com.squareup.sqldelight.Transacter
+import com.squareup.sqldelight.db.SqlDatabase
+import com.squareup.sqldelight.db.SqlDatabaseConnection
+import com.squareup.sqldelight.db.SqlPreparedStatement
+import com.squareup.sqldelight.db.SqlCursor
+import com.squareup.sqldelight.db.SqlPreparedStatement.Type.DELETE
+import com.squareup.sqldelight.db.SqlPreparedStatement.Type.EXECUTE
+import com.squareup.sqldelight.db.SqlPreparedStatement.Type.INSERT
+import com.squareup.sqldelight.db.SqlPreparedStatement.Type.SELECT
+import com.squareup.sqldelight.db.SqlPreparedStatement.Type.UPDATE
+
+class SQLiterHelper(private val databaseManager: DatabaseManager) : SqlDatabase {
```

Comment:
@kpgalligan the biggrest change i made here was to just have a single connection total. The generated code expects`getConnection()` calls to return the same connection per thread, and since we're single threaded frozen I figure this is safe for now. Can revisit later